### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by pierrejeambrun**
Redirect reporters to the airflow-s fork with Private Vulnerability Reporting

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -29,15 +29,13 @@ to learn more, head to the
 The Apache Software Foundation takes security issues very seriously. Apache
 Airflow specifically offers security features and is responsive to issues
 around its features. If you have any concern around Airflow Security or believe
-you have uncovered a vulnerability, we suggest that you get in touch via the
-e-mail address [security@airflow.apache.org](mailto:security@airflow.apache.org).
+you have uncovered a vulnerability, we suggest that you privately report it by using the [vulnerability reporting form](https://github.com/airflow-s/airflow/security/advisories/new).
```
Comment (hussein-awala) [inline .github/SECURITY.md:32]: I thought that the other fork was just for testing

Comment (hussein-awala) [reply .github/SECURITY.md:32]: But yeah, I think it's better to keep everything related to the security there.

**Review by hussein-awala — APPROVED** [review]

File: .github/SECURITY.md (hunk 1 above)
Comment (pierrejeambrun) [reply .github/SECURITY.md:32]: In order to test things I figured we need people to use that new way of reporting, and redirecting them would be a good start.

We can also redirect when we receive emails at security@airflow.apache.org/private@airflow.apache.org

**Comment by pierrejeambrun** [comment]
Converting to draft following Jed comments. Will update if the experiement is succesfull and that Private Vulnerability Reporting is enabled on apache/airflow

**Comment by pierrejeambrun** [comment]
up

**Pull request closed without merging**